src/validation.cpp

```diff
@@ -342,11 +342,14 @@ static bool IsCurrentForFeeEstimation(CChainState& active_chainstate) EXCLUSIVE_
  * Passing fAddToMempool=false will skip trying to add the transactions back,
  * and instead just erase from the mempool as needed.
  */
-
-static void UpdateMempoolForReorg(CChainState& active_chainstate, CTxMemPool& mempool, DisconnectedBlockTransactions& disconnectpool, bool fAddToMempool) EXCLUSIVE_LOCKS_REQUIRED(cs_main, mempool.cs)
+void CChainState::UpdateMempoolForReorg(
```

Comment:
Mixed feelings about this change. I think a reader might get confused: "`UpdateMempoolForReorg`? But I don't have mempool."

I would slightly prefer changing the name to `UpdateMempoolForReorgIfExists` or `TryUpdateMempoolForReorg`, but feel free to ignore.

Reply:
Sounds good, fixed.